CMake/Dependencies/libkvspic-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvspic-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-pic.git
-	GIT_TAG           ff1ba18247355f9e99cc1c482ae12d91ced8d059
+	GIT_TAG           9a52a1005988d2a3521e23f6ef8505d0034fd329
```

Comment:
This is the issue, this is the commit from master, master doesn't have your changes yet.